test/integration/exceptionalOpeningTimes.js

```diff
@@ -18,38 +18,55 @@ function mockCurrentDate2() {
 }
 
 function expectExceptionalOpeningTimes1($, rows, times) {
-  expect($(rows[0]).text()).to.include('Monday 29 May');
+  expect($(rows[0]).text()).to.include('Thursday 25 May');
   expect($(rows[0]).text()).to.include(times[0]);
+  expect($(rows[1]).text()).to.include('Monday 29 May');
+  expect($(rows[1]).text()).to.include(times[1]);
 }
 
 function expectExceptionalOpeningTimes2($, rows, times) {
+  expect($(rows[0]).text()).to.not.include('Thursday 25 May');
+  expect($(rows[0]).text()).to.not.include('8am to 12pm');
   expect($(rows[0]).text()).to.include('Monday 29 May');
   expect($(rows[0]).text()).to.include(times[0]);
```

Comment:
Could be simplified by replacing times[0] with 'Closed' and removing the times parameter